01-node-tutorial/answers/customEmitter.js

```diff
@@ -0,0 +1,18 @@
+const EventEmitter = require("events");
+
+const customEmitter = new EventEmitter();
+
```

Comment:
This does not use promises or subinterval. Please re-read the directions.

Reply:
code has been rewritten with use of Promise and setInterval.